StPicoDstMaker/StPicoDst.h

```diff
@@ -21,8 +21,9 @@ class StPicoMtdPidTraits;
 class StPicoDst : public TObject
 {
 public:
-  /// constructor
-  StPicoDst();
+  StPicoDst() {}
+  virtual ~StPicoDst() {}
```

Comment:
I agree that there is nothing polymorphic in the design. The class is forced to be polymorphic by the virtual ::Print() method, I was hesitant to change it since it seems to be the prevalent choice for Print in STAR (https://github.com/star-bnl/star-muDst/blob/master/StMuDSTMaker/COMMON/StMuDst.h#L432). 

I just wanted to get rid of the compiler warning about the dtor not being virtual.